src/libraries/Utils.sol

```diff
@@ -12,4 +12,40 @@ library Utils {
     function isFeatureEnabled(string memory _feature) internal view returns (bool) {
         return vm.envOr(_feature, false) || vm.envOr(_feature, uint256(0)) == 1;
     }
+
+    /// @notice Checks that values have code on this chain.
+    /// This method is not storage-layout-aware and therefore is not perfect. It may return erroneous
+    /// results for cases like packed slots, and silently show that things are okay when they are not.
+    function isLikelyAddressThatShouldHaveCode(uint256 value, address[] memory codeExceptions)
```

Comment:
Do we need a test for this function? I'm not sure, since we are already warning that it might not always work.

Reply:
No worries, I saw it's not a new function.  We can let it be.